app/components/forms/session-speaker-form.js

```diff
@@ -75,11 +81,24 @@ export default Component.extend(FormMixin, {
           ]
         },
         sessionType: {
+          identifier : 'session_sessionType',
+          rules      : [
+            {
+              type   : 'isTrackField',
+              prompt : this.l10n.t('Please select a Track Associated first')
+            }
+          ]
+        },
+        sessionTypeReq: {
           identifier : 'session_sessionType_required',
           rules      : [
             {
               type   : 'empty',
               prompt : this.l10n.t('Please select a session type')
+            },
+            {
+              type   : 'isTrackField',
```

Comment:
@niranjan94 @CosmicCoder96 The following validation is not working , ?

Reply:
Anything you can help with

Reply:
@niranjan94 @CosmicCoder96 I am not able to figure out, Why the `Session Track` validations fail when I fill in the `Session Type`.
I have already added validation to check if `Track` is non-empty but filling the `Session_Type` bypasses that validation. Please see if you can figure out something. Or should I close the PR